ClipboardMonitor/NativeMethods.txt

```diff
@@ -0,0 +1,12 @@
+AddClipboardFormatListener
+SetParent
+GetWindowText
+GetWindowTextLength
+GetForegroundWindow
+GetWindowThreadProcessId
+GetCurrentProcess
+
+// Below methods do not work properly with or does not exist in CsWin32
+//GetKernelObjectSecurity
+//SetKernelObjectSecurity
+//NtSetInformationProcess
```

Comment:
`NtSetInformationProcess` is indeed missing from the metadata, as is `SetInformationProcess`. @mikebattista, any idea why?